bundles/automation/org.eclipse.smarthome.automation.rest/src/main/java/org/eclipse/smarthome/automation/rest/internal/ModuleTypeResource.java

```diff
@@ -58,13 +62,16 @@
  */
 @Path("module-types")
 @Api("module-types")
+@Component(service = { RESTResource.class, ModuleTypeResource.class })
```

Comment:
Why do you declare and so export the class itself as provided service?

Reply:
Changed because of [this](https://github.com/eclipse/smarthome/pull/5265#discussion_r188215435)